### ISSUE TITLE
bash: epr: command not found

### ISSUE THREAD

**Issue by aadilayub** [wustho/epr#11]
I've installed epr just now and can't seem to get it to work. Here's my terminal output:

![image](https://user-images.githubusercontent.com/31581044/58826499-47207980-865a-11e9-8055-13ff5ff14be1.png)

Not sure if it's just something wrong with my local environment

**Comment by wustho** [comment]
Hello there, sorry just read this, I'm on the other side of your time zone. Firstly, thanks for giving full walkthrough of your problem. TLDR at bottom.

From what I can see, this `epr` is not available in pypi, so when you run `pip install epr`, you install another `epr` package which is not this one... so if you want to install this `epr`, you need to pip install from github repo that is by this kinda long command: `pip install git+https://github.com/wustho/epr.git@commontoc`. _But_ since you already install another epr from pypi, you need to uninstall it first before install this epr.

TL;DR: try running this command and see if the problem still persists

`pip uninstall epr & pip install git+https://github.com/wustho/epr.git@commontoc`

**Comment by aadilayub** [comment]
Thanks for your detailed answer! It worked
